Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -578,15 +578,14 @@ AppDelegateViewModelOutputs {
       self.userSessionStartedProperty.signal,
       self.userSessionEndedProperty.signal
       )
+      .filter { !AppEnvironment.current.mainBundle.isDebug }
```

Comment:
So just to confirm - this line would mean HockeyApp doesn't run on debug builds at all (the ones we're testing locally on device) which I think is fine.

I would just say that we probably don't need the additional check on line `588` as we should trust the `filter` to prevent that `mainBundle.isDebug` would ever be `true` here.